### ISSUE TITLE
Could not find any implementation for node

### ISSUE THREAD

**Issue by jiajihao** [NVIDIA/TensorRT#2918]
## Description

E] Error[10]: [optimizer.cpp::computeCosts::3728] Error Code 10: Internal Error (Could not find any implementation for node {ForeignNode[177...Reshape_81 + (Unnamed Layer* 146) [Shuffle]]}.)
[04/28/2023-14:06:47] [E] Error[2]: [builder.cpp::buildSerializedNetwork::751] Error Code 2: Internal Error (Assertion engine != nullptr failed. )

**command:** ./trtexec --onnx="m2a_net.onnx" --saveEngine="test.engine"

The model of onnx can be inferred normally using onnxruntime

## Environment

**TensorRT Version**:8.5.2.2 
**NVIDIA GPU**: RTX 2080
**NVIDIA Driver Version**: 455.45.01
**CUDA Version**: 11.1
**CUDNN Version**: 8.8.1.3 for cuda11
**Operating System**: ubuntu 18.04
**Python Version (if applicable)**: 3.6&2.7(default)
**Tensorflow Version (if applicable)**: x
**PyTorch Version (if applicable)**: 2.1.0
**Baremetal or Container (if so, version)**: x

## Relevant Files
https://drive.google.com/file/d/1S92YumvKtQhAGKSc-8aYXbBNNr6772AD/view?usp=share_link

**Comment by zerollzeng** [comment]
You driver version is pretty old. Could you please try with our latest official TensorRT docker?

**Comment by zerollzeng** [comment]
TRT has it's own op specification and we list them on the api doc. E.g. https://docs.nvidia.com/deeplearning/tensorrt/api/c_api/classnvinfer1_1_1_i_scatter_layer.html

**Comment by ttyio** [comment]
Closing since no activity for more than 3 weeks, thanks!
